bodhi-server/bodhi-server.spec

```
@@ -157,11 +130,6 @@ install -pm0644 docs/_build/*.1 %{buildroot}%{_mandir}/man1/
 %files -n bodhi-composer
 %license COPYING
 %doc README.rst
```

This suggestion got lost somehow, so sending it separately.

```suggestion
%doc README.rst
%pycached %{python3_sitelib}/bodhi/server/tasks/composer.py
%pycached %{python3_sitelib}/bodhi/server/tasks/metadata.py
```